code/game/objects/items/rogueweapons/melee/blunt.dm

```diff
@@ -354,6 +354,8 @@
 	icon_state = "polemace"
 	force = 15
 	force_wielded = 35
+	smeltresult = /obj/item/ingot/steel
+	smelt_bar_num = 2
```

Comment:
This var is in a lot of places, I'd rather keep everything at 1 smelting result because this makes it so if we're not careful while adding a new weapon or armor we can introduce map duping by mistake.

I also don't think weapons should give their costs 1:1 when smelting!

Reply:
i was honestly iffy about it too. honestly fuck that shit